media_driver/linux/common/ddi/media_libva_util.cpp

```diff
@@ -405,6 +411,23 @@ VAStatus DdiMediaUtil_AllocateSurface(
     uint32_t    gmmPitch;
     uint32_t    gmmSize;
     uint32_t    gmmHeight;
+
+    gmmPitch    = (uint32_t)gmmResourceInfo->GetRenderPitch();
+    if( DdiMediaUtil_IsExternalSurface(mediaSurface) && ( mediaSurface->pSurfDesc->uiVaMemType == VA_SURFACE_ATTRIB_MEM_TYPE_USER_PTR ) )
+    {
+        uint32_t gmmHAlign;
+        gmmHAlign = gmmResourceInfo->GetHAlign();
```

Comment:
I think this better be on the single line as:
int32_t gmmHAlign = gmmResourceInfo->GetHAlign();

Reply:
sure